recipe/test_blas.sh

```diff
@@ -21,38 +21,12 @@ fi
 if [[ "$target_platform" == osx-* ]]; then
   # testing with shared libraries does not work. skip them.
   # to test that program exits if wrong parameters are given, what the testsuite
-  # do is that the symbol xerbla (xerbla logs the error and exits) is overriden
+  # does is that the symbol xerbla (which logs the error and exits) is overriden
   # by the test program's own version which reports to the test program that
   # xerbla was called. This does not work with dylibs on osx and dlls on windows
   SKIP_TESTS="${SKIP_TESTS}|x*cblat2|x*cblat3"
 fi
 
-if [[ "${blas_impl}" == "mkl" ]]; then
```

Comment:
Why remove these?

Reply:
This was more exploratorily, because I wanted to know which other tests might be failing (especially since the comments say `Skip for now` without much more context), resp. to come to a more substantiated decision whether it might be justified to include these new failures in the skips.

Also, not nearly all of the tests that were skipped here are actually failing. So I think it's a good idea to clean up this list a bit.

Reply:
Okay, so the test failures you mentioned were not new failures?

Reply:
Oh no, the test failures I mentioned were indeed new, but then I had a look at `test_blas.sh` and decided to find out how the situation looks without all the skips.

Reply:
See analysis [here](https://github.com/conda-forge/blas-feedstock/pull/55#issuecomment-751893340) and ensuing discussion.